src/resources/styles-data.ts

```diff
@@ -178,8 +178,6 @@ export const derivedStyles = {
   "slider-track-color": "var(--scrollbar-thumb-color)",
   "label-badge-background-color": "var(--card-background-color)",
   "label-badge-text-color": "rgba(var(--rgb-primary-text-color), 0.8)",
-  "paper-listbox-background-color": "var(--card-background-color)",
-  "paper-item-icon-active-color": "var(--state-icon-active-color)",
```

Comment:
Good catch :+1:

These are also used in the demo themes. Please remove it there too